.travis.yml

```diff
@@ -40,13 +40,16 @@ before_script:
 - git clone https://$MC_DEV_ADMIN_GH_TOKEN@github.com/masschallenge/front-end -b $FRONT_END_REVISION ../front-end
 - cp .prod.env ../front-end/.env
 - cp .prod.env .env
+- ACCELERATE_VERSION=$(git describe --tags --abbrev=0)
+- export ACCELERATE_VERSION
+- echo $ACCELERATE_VERSION
```

Comment:
I'd rather not add more output to the (already very noisy) Travis log, but if we include this let's do something like:
 `echo "ACCELERATE_VERSION: $ACCELERATE_VERSION"`

Reply:
sure, I was testing the `git` command if it had got the latest tag. It will not be existing in the final. The remaining bit is figuring the testing instructions. Since we cannot `ssh` into the containers, hoping to use cloudwatch since it also needs to be tested for production environments.